ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/ViewAction.java

```diff
@@ -112,6 +112,10 @@ public void openAddAndRemoveDialog(String serverLabel) {
 		_serversView.getServers().contextMenu(ADD_AND_REMOVE, serverLabel);
 	}
 
+	public void openFileMenuFragmentFilesWizard() {
```

Comment:
1. I saw you add "openLiferayModuleFragmentFilesWizard" this method into wizardAction, what do you think put these two in on *Action class?